src/dom.js

```diff
@@ -216,64 +186,11 @@ var dom = module.exports = {
 	/**
 	* @private
 	*/
-	// Workaround for lack of compareDocumentPosition support in IE8
-	// Code MIT Licensed, John Resig; source: http://ejohn.org/blog/comparing-document-position/
-	compareDocumentPosition: function(a, b) {
```

Comment:
Spotlight uses `dom.compareDocumentPosition()` so we'll need to either keep a facade here or update spotlight.

https://github.com/enyojs/spotlight/blob/2.6.0-dev/src/container.js#L278